tests/python/unittest/test_sparse_operator.py

```diff
@@ -697,6 +697,19 @@ def check_binary_op_with_scalar(stype,
                                        force_overlap=force_overlap,
                                        verbose=False)
 
+        # plus_scalar
```

Comment:
Do we have tests for scatter_plus/scatter_div?

Reply:
Yes

Reply:
Also scatter_minus